handlers/update-supporter-plus-amount/test/fixtures/subscription.json

```
@@ -0,0 +1,398 @@
+{
+  "success": true,
+  "id": "8ad084db8ec66344018ec793fed67828",
+  "accountId": "8ad09be48b6b4f7d018b6d422dfb35ad",
+  "accountNumber": "A00714188",
+  "accountName": "0019E00002NxBIOQA3",
+  "invoiceOwnerAccountId": "8ad09be48b6b4f7d018b6d422dfb35ad",
+  "invoiceOwnerAccountNumber": "A00714188",
+  "invoiceOwnerAccountName": "0019E00002NxBIOQA3",
+  "subscriptionNumber": "A-S00707842",
+  "version": 2,
+  "revision": "3.0",
+  "termType": "TERMED",
+  "invoiceSeparately": false,
+  "contractEffectiveDate": "2023-10-26",
+  "serviceActivationDate": "2023-10-26",
+  "customerAcceptanceDate": "2023-10-26",
+  "subscriptionStartDate": "2023-10-26",
+  "subscriptionEndDate": "2025-04-10",
+  "lastBookingDate": "2024-04-10",
+  "termStartDate": "2024-04-10",
+  "termEndDate": "2025-04-10",
+  "initialTerm": 167,
+  "initialTermPeriodType": "Day",
+  "currentTerm": 12,
+  "currentTermPeriodType": "Month",
+  "autoRenew": true,
+  "renewalSetting": "RENEW_WITH_SPECIFIC_TERM",
+  "renewalTerm": 12,
+  "renewalTermPeriodType": "Month",
+  "currency": "USD",
+  "contractedMrr": 12.08,
+  "totalContractedValue": 167.85,
+  "notes": "Updated amount from Postman to mimic MMA",
+  "status": "Active",
+  "TrialPeriodPrice__c": null,
+  "CanadaHandDelivery__c": null,
+  "AcquisitionMetadata__c": null,
+  "QuoteNumber__QT": null,
+  "GifteeIdentityId__c": null,
+  "OpportunityName__QT": null,
+  "LastQSSPaymentDate__c": null,
+  "GiftNotificationEmailDate__c": null,
+  "Gift_Subscription__c": "No",
+  "TrialPeriodDays__c": null,
+  "CreatedRequestId__c": "4bf79241-025f-4156-0000-000000001a54",
+  "ActivationDate3__c": null,
+  "AcquisitionSource__c": null,
+  "CreatedByCSR__c": null,
+  "CASSubscriberID__c": null,
+  "LastPriceChangeDate__c": null,
+  "InitialPromotionCode__c": null,
+  "Suspended__c": "false",
+  "CpqBundleJsonId__QT": null,
+  "RedemptionCode__c": null,
+  "QuoteType__QT": null,
+  "GiftRedemptionDate__c": null,
+  "QuoteBusinessType__QT": null,
+  "SupplierCode__c": null,
+  "legacy_cat__c": null,
+  "DeliveryAgent__c": null,
+  "AcquisitionCase__c": null,
+  "ReaderType__c": "Direct",
+  "ActivationDate__c": null,
+  "UserCancellationReason__c": null,
+  "SuspensionStatus__c": "Active",
+  "CardCountry__c": null,
+  "OpportunityCloseDate__QT": null,
+  "IPaddress__c": null,
+  "IPCountry__c": null,
+  "CancelledBy__c": null,
+  "dummy__c": null,
+  "PromotionCode__c": null,
+  "OriginalSubscriptionStartDate__c": null,
+  "LegacyContractStartDate__c": null,
+  "CancellationReason__c": null,
+  "billToContact": {
+    "id": "8ad09be48b6b4f7d018b6d422e3835bc",
+    "address1": "Kings Place - giftee",
```

probably delete that to reduce confusion
```suggestion
    "address1": "Kings Place",
```